plugins/tmdb.py

```diff
@@ -0,0 +1,108 @@
+from collections import OrderedDict
+from urllib.parse import quote_plus
+from . import plugin
+
+import requests
+
+
+class TMDBPlugin(plugin.Plugin):
+    def __init__(self):
+        super().__init__()
+        self._TMDB_API_URL = self._config.get("TMDB_API_URL")
+        self._TMDB_API_KEY = self._config.get("TMDB_API_KEY")
+        self._TMDB_MAX_RESULTS = self._config.get("TMDB_MAX_RESULTS", 3)
+
+    def receive(self, request):
+        if super().receive(request) is False:
+            return False
+
+        if request['text'].lower().startswith("moonbeam movies"):
+            self._log.debug(f"Got movie request: {request['text']}")
+            command = request['text'].split()
+
+            movie = ' '.join(command.split()[2:])
```

Comment:
You already `split()` the incoming text string in L21, so you don't need to (and can't) do it again here.

Reply:
D'oh, you are correct sir